packages/testing/src/timeouts.ts

```diff
@@ -102,19 +103,27 @@ export function adjustCurrentTestTimeout(ms: number) {
 }
 
 /**
+ * @deprecated use {@link debugSafeTimeout} instead
  * Creates a playwright locator options with {@link scaleTimeout| scaled } timeout
  * and adjust the current test timeout accordingly
  */
 export function locatorTimeout(ms = 10_000) {
-    return { timeout: adjustCurrentTestTimeout(scaleTimeout(ms)) };
+    const timeout = scaleTimeout(ms);
+    adjustCurrentTestTimeout(timeout);
+    return setAdjustedTimeout({ timeout });
```

Comment:
Just a thought — what about the name of the method here, `markAdjustedTimeout`? 
Non-blocking suggestion, though